### PR TITLE
SPM-674 Add Clowder support 1 (db-admin, manager)

### PR DESCRIPTION
**Pull request by Josca**
- Basic Clowder support works.
- Clowder params are exported to environment vars using `try_export_clowder_params.sh`.
- Two db-admin vars had to be renamed (DB_USER -> DB_ADMIN_USER, DB_PASSWD -> DB_ADMIN_PASSWD), to be consistent with Clowder params.
- Added `deploy/clowdapp.yml` template to deploy Patchman using Clowder to Ephemeral environemnt - used in new `ci.ext.devshift.net` PR pipeline. Only database-admin and manager for now.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: database_admin/update.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-export PGUSER=$DB_USER
-export PGPASSWORD=$DB_PASSWD
+export PGUSER=$DB_ADMIN_USER
+export PGPASSWORD=$DB_ADMIN_PASSWD
 export PGDATABASE=$DB_NAME
 export PGPORT=$DB_PORT
 export PGSSLMODE=$DB_SSLMODE
 
-WAIT_FOR_EMPTY_DB=1 ./scripts/wait-for-services.sh
+DB_USER=$DB_ADMIN_USER DB_PASSWD=$DB_ADMIN_PASSWD WAIT_FOR_EMPTY_DB=1 ./scripts/wait-for-services.sh
```
Comment (MichaelMraka) [inline database_admin/update.sh:10]: Would not be better to update `wait-for-services.sh` to accept `DB_ADMIN_*` params?

File: scripts/try_export_clowder_params.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+function print_clowder_params() {
+  if [[ -n $GORUN ]]; then
+    go run main.go print_clowder_params
+  else
+    ./main print_clowder_params
+  fi
+}
+
+# Detect params, it should be printed in 'PARAM=VALUE' format
+CLOWDER_PARAMS=$(print_clowder_params | grep '=' || true) # use '|| true' not to stop on empty match
```
Comment (MichaelMraka) [inline scripts/try_export_clowder_params.sh:14]: `|| true` is redundant
`CLOWDER_PARAMS=$(print_clowder_params | grep '=' )` will not fail even if there's no `=` and script runs with `set -e`

File: evaluator/metrics.go [hunk 3]
```diff
@@ -64,7 +65,9 @@ func RunMetrics(port string) {
 	core.InitProbes(app)
 	middlewares.Prometheus().Use(app)
 
-	err := utils.RunServer(base.Context, app, ":"+port)
+	port := utils.GetIntEnvOrDefault("PORT", 8082)              // legacy env var
+	publicPort := utils.GetIntEnvOrDefault("PUBLIC_PORT", port) // clowder env var
+	err := utils.RunServer(base.Context, app, fmt.Sprintf(":%d", publicPort))
```
[the remark this answers is not in the document]
Comment (Josca) [reply evaluator/metrics.go]: Good point, updated.

File: scripts/try_export_clowder_params.sh (hunk 2 above)
Comment (MichaelMraka) [reply scripts/try_export_clowder_params.sh:14]: ok, taking back. it's correct

**Review by MichaelMraka — APPROVED** [review]

**Review by semtexzv — APPROVED** [review]

**Pull request merged**